MBC_userImport.class.inc

```diff
@@ -69,7 +69,7 @@ class MBC_UserImport
   /**
    * The number of queue entries to process in each session
    */
-  const BATCH_SIZE = 5000;
+  const BATCH_SIZE = 5;
```

Comment:
Is there a reason why you dropped the batch size so dramatically? Maybe forgot after tests?